tests/page-objects/common/common.wdio.page.js

```diff
@@ -0,0 +1,59 @@
+const hamburgerMenu = () => $('#header-dropdown-link');
+const logoutButton =  () => $('.fa-power-off');
+const modalBody = () => $('div.modal-body');
+const yesButton = () => $('a.btn.submit.btn-danger');
```

Comment:
I don't know our modal css enough but I wonder if we could pull the modal out into it's own object for re-use.

Reply:
The structure of the modal is: 
![image](https://user-images.githubusercontent.com/35681649/126482081-c9c127da-7874-4bba-af27-431ffa9883ad.png)

Ignoring the actual modal that we have here, we always have: 
```
> modal-container
  > div.modal-dialog
    > div.modal-content
      > <the actual modal element, varies by modal type>
          > div 
            >  div.modal-header <- contains modal title + the "x" close button
            >  div.modal-body <- contains modal text
            >  div.modal-footer <- contains modal buttons
```

if we want to make sure we're clicking a button from a modal, we can prefix the selector with `.modal-dialog` or `.modal-content` or, even stricter, `.modal-footer`. Either one works for me.

Reply:
Thanks for clarifying. I put the modal into it's own object and when using it will be 

```
const modal = require('./modal.wdio.page.js')

await modal.confirm()
await modal.body()
etc...
````

Reply:
> await modal.confirm();

Looks great! It makes the action so much more obvious!